src/toil/test/src/jobCacheTest.py

```diff
@@ -350,7 +350,8 @@ def _parseAssertionError(logFile):
                 for line in logFileHandle:
                     line = line.strip()
                     if workerLogName is None:
-                        if line.startswith('Reporting'):
+                        if line.startswith('The jobWrapper seems to have left '
```

Comment:
Not happy about this duplication. How did you find out about this? Is this part crucial for the test?